### ISSUE TITLE
[Search] Workshop is not found when location is removed 

### ISSUE THREAD

**Issue by SyrvatkaTetiana** [ita-social-projects/OoS-Backend#1416]
**Environment:** Windows 11 Home, browser: Google Chrome 121.0.6167.189.
**Reproducible:** Always
**Build found:**  "commit [a76b5f7](https://github.com/ita-social-projects/OoS-Backend/commit/a76b5f70ec8343f9891ae57f2f21979b4e01cc57)"

**Preconditions**
1. The site https://oos.dmytrominochkin.cloud/#/ is opened

**Steps to reproduce**
1. Clear location in the search field
2. Enter name of the workhop in the serch field (e.g: Спляча Красуня)
3. Press 'Enter' 

**Actual result**
No reults found

**Expected result**
Workshop is found

**User story and test case links**
"User story #838 

<img width="837" alt="image" src="https://github.com/ita-social-projects/OoS-Backend/assets/151456891/791ecc35-b81e-4966-b60c-47659c1d8dbe">

**Comment by olsavchenk** [comment]
@SyrvatkaTetiana I have tried to reproduce this error, but without success. When I clear the location field and press "Enter", the location is automatically set to my current location. I also tested this with location data turned off in my browser and this case also failed, the location was automaticaly set to Kyiv.
https://www.loom.com/share/d37b2fa4fa1449e9a429e40e9f180cb3?sid=a63f79f4-6895-46a0-a1a5-17c0cbb01c6c
